### PR TITLE
docs: adjust readme to better understandment about what the pluign is

### PR DESCRIPTION
**Pull request by miguel91it**

### CODE REVIEW

**Review by alvarowolfx — COMMENTED** [review]
Added some minor typos and comments.

File: README.md [hunk 1]
```diff
@@ -17,27 +29,26 @@ Here are the steps to configure and use the WebSocket Data Source Plugin in Graf
    - Scroll to the bottom and chose "WebSocket API" in the "Others" category.
 
 2. Configure WebSocket endpoint information
-
-   ![Grafana Websockets Configuration](https://user-images.githubusercontent.com/8334211/150139294-8a374964-b9a4-40c4-8cce-7b4b32d21f69.png)
+   ![image](https://user-images.githubusercontent.com/8334211/155154644-8925b616-a5e0-4c32-92bd-305696d0a4d1.png)
```
[the remark this answers is not in the document]
Comment (miguel91it) [reply README.md:32]: To have first a text with a better explanation about what the plugin is. The image at begining directly doesn't make sense and the image itself already was being used in the third item. So, it was being redundant.

[the remark this answers is not in the document]
Comment (miguel91it) [reply README.md:32]: Forgotten this.

**Comment by miguel91it** [comment]
@alvarowolfx, fixed!

**Review by alvarowolfx — APPROVED** [review]

**Pull request merged**